src/SQLChecks/Get-AutoGrowthRisks.ps1

```diff
@@ -0,0 +1,15 @@
+﻿Function Get-AutoGrowthRisks {
+    [cmdletbinding()]Param(
+         [parameter(Mandatory=$true)][string]$ServerInstance
+        ,$WhitelistFilegroups # optional array or comma-delim string
+    )
+
+    $WLFGNames=@()
+    if($WhitelistFilegroups -ne $null){$WLFGNames+=$WhitelistFilegroups.Split(",")}
+
+    $query=(gc $PSScriptRoot/../SQLScripts/auto-growth-will-fail-for-max_size.sql -Raw)

```

Comment:
Does this line make the command non-portable (e.g. if I happen to be in a different folder...).  If so would rather put the SQL inline.

Reply:
I *believe* I tested & this is not a binding design. It's also helpful since you can quickly toggle the `.sql` file and modify the pester without re-importing the module/re-defining the command. I'll run a few quick re-checks on this though. 

Keeping the `.sql` file separate makes it useable more quickly as a separate script when pester returns **non-zero bad**. It also segments the code in a way that is pleasing to me personally :trollface: ...but as always, I'm happy to correct it if it's a bad design.

Reply:
Confirmed relative directory location isn't breaking. 

```powershell
cd c:\
Get-FixedSizeFileGroups -ServerInstance localhost -WhitelistFilegroups $ZeroAutoGrowthWhitelistFGs
```

Reply:
That's cool, and good to know.

Still thinking on this but my gut is I'm going to refactor this to inline SQL to match the rest of the scripts (having a .sql file lying around means its then easy to break your function by modifying the return values of the script, and means you have to change two files to modify that functionality).